Cargo.toml

```diff
@@ -21,6 +21,19 @@ volatile-register = "0.2.0"
 bitfield = "0.13.2"
 embedded-hal = "0.2.4"
 
+[features]
+cm7-r0p1 = []
+inline-asm = []
+
+# Link against prebuilt assembly blobs that are compatible with Linker-Plugin LTO.
+# When using this feature, you *must* pass `-Clinker-plugin-lto` to rustc. See this link for
+# details:
+# https://doc.rust-lang.org/stable/rustc/linker-plugin-lto.html
+linker-plugin-lto = []
```

Comment:
Could you also add that it isn't possible to use this feature on opt-level s/z ?

Reply:
Moved the docs into lib.rs and expanded them